Dockerfile

```diff
@@ -3,6 +3,7 @@
 #     CGO_ENABLED=0 go build -a -tags netgo
 #     docker build --rm=true -t plugins/drone-cache .
 
-FROM gliderlabs/alpine:3.1
+FROM alpine:3.4
```

Comment:
I don't know why we need Alpine for all plugin.

`centurylink/ca-certs` is also base image on drone/drone repo.

https://github.com/drone/drone/blob/master/Dockerfile